can_messages.asciipb

```diff
@@ -182,17 +183,7 @@ msg {
   }
 }
 
-msg {
-  id: 19
```

Comment:
How are you planning on providing feedback to the driver when they are engaging Cruise Control?

How is whatever is driving your driver display going to get this data?

Reply:
@karlding since drive output already packs the cruise_target data and sends it to the motor controllers. I could just update driver display (in the telemetry repo) to display this data instead of the previous cruise_target CAN message?

Reply:
turns out that driver display already uses the cruise_target from the drive output message to display the data https://github.com/uw-midsun/telemetry/blob/master/client/src/ts/dd_main.ts